Documentation/guides/building-apps/build-items.md

```diff
@@ -213,6 +240,49 @@ step).
 Starting in Xamarin.Android 5.1, attempting to use the `@(Content)`
 Build action will result in a `XA0101` warning.
 
+## EmbeddedJar
+
+In a Xamarin.Android binding project, the **EmbeddedJar** build action
+binds the Java/Kotlin library and embeds the `.jar` file into the
+library. When a Xamarin.Android application project consumes the
+library, it will have access to the Java/Kotlin APIs from C# as well
+as include the Java/Kotlin code in the final Android application.
+
+Since Xamarin.Android 11.2, you can use the **AndroidLibrary**
+build action as an alternative.
+
+## EmbeddedNativeLibrary
+
+In a Xamarin.Android class library or Java binding project, this
+bundles a native library such as `lib/armeabi-v7a/libfoo.so` into the
+library. When a Xamarin.Android application consumes the library, the
+`libfoo.so` file will be included in the final Android application.
+
+Since Xamarin.Android 11.2, you can use the **AndroidNativeLibrary**
```

Comment:
This can use a link:

```markdown
Since Xamarin.Android 11.2, you can use the [**AndroidNativeLibrary**](#androidnativelibrary)
build action as an alternative.
```